header.go

```diff
@@ -113,10 +113,10 @@ func ParseAddressList(list string) ([]*mail.Address, error) {
 //   - Missing ';' between content-type and media parameters
 //   - Repeating media parameters
 //   - Unquoted values in media parameters containing 'tspecials' characters
-func ParseMediaType(ctype string) (mtype string, params map[string]string, invalidParams []string,
```

Comment:
Instead of changing the signature of this function, let's just have it default to not stripping the invalid characters, and deprecate the entire function instead.  It's redundant now that our `mediatype` is a public package.